ios/Classes/FreshchatSdkPlugin.m

```diff
@@ -377,6 +383,7 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
         result([instance getUserIdTokenStatus]);
     }else if([@"identifyUser" isEqualToString:call.method]){
         [instance identifyUser:call];
+        result(nil);
```

Comment:
`identifyUser` should not be a future.
It was a mistake. We will correct it.
You can revert setting result.